src/components/Modal.js

```diff
@@ -0,0 +1,46 @@
+import React, {useState} from 'react';
+import Modal from 'react-modal';
+import CountySelector from './countySelector';
+
+//Styles for Modal with county Dropdown//
+const customStyles = {
+    content : {
+      top: '50%',
+      left: '50%',
+      right: 'auto',
+      bottom: 'auto',
+      marginRight           : '-50%',
+      transform: 'translate(-50%, -50%)',
+      backgroundColor: 'green',  
+      zIndex: '10',  
+      height:  "150px",
+      width: "80vw",
+    }
+};
+
+//-----------Function to return Dropdown Modal-------//
+function ModalInFunctionalComponent (){
+
+    const [modalIsOpen,setModalIsOpen] = useState(false);
+
+    const setModalIsOpenToTrue =()=>{
+        setModalIsOpen(true)
+    }
+
+    const setModalIsOpenToFalse =()=>{
+        setModalIsOpen(false)
+    }
```

Comment:
This is a bit excessive for a function name, and for the functionality. You could have easily combined this into one function and given it the name of `modalHandler` or such